rundbcmd.py

```diff
@@ -0,0 +1,440 @@
+    """Given bytes and units ( K, M, G, T)
+       convert input bytes to that unit:
+             vtemp = convert_size(float(vtemp),"M")
+    """
+
+    size_bytes = arg_size_bytes
+
+    if size_bytes == 0:
+       return "0B"
+
+    vunit = vunit.upper()
+
+    size_name = ("B", "K", "M", "G", "T", "P", "E", "Z", "Y")
+
+    vidx = int(size_name.index(vunit))
+
+    p = math.pow(1024, vidx)
+    s = round(size_bytes / p, 2)
+    # i = int(math.floor(math.log(size_bytes, 1024)))
+    # p = math.pow(1024, i)
+    # s = round(size_bytes / p, 2)
+    return "%s%s" % (int(round(s)), size_name[vidx])
+
+
+def israc(host_str=None):
+    """
+    Determine if a host is running RAC or Single Instance
+    """
+    global err_msg
+    global cru_domain
+    global dr_domain
+
+    if host_str is None:
+        return()
+
+    if "org" in host_str:
+        host_str = host_str.replace(cru_domain,"")
+        host_str = host_str.replace(dr_domain, "")
+
+    if "dr" in host_str or "dw" :
+        return(False)
+
+    # if the last digits is 1 or 2 ( something less than 10) and not 0 (60) return True
+    if host_str[-1:].isdigit() and int(host_str[-1:]) < 10 and int(host_str[-1:]) != 0:
+        return(True)
+    else:
+        return(False)
+
+
+def run_cmd(cmd_str):
+    """
+    Encapsulate all error handline in one fx. Run cmds on the local machine.
+    """
+
+    try:
+        process = subprocess.Popen([cmd_str], stdout=PIPE, stderr=PIPE, shell=True)
+        output, code = process.communicate()
+    except:
+        pt1 = "Error :: run_cmd() :: running cmd_str: {}".format(cmd_str)
+        add_to_msg(pt1)
+        pt2 = " {}, {}, {}".format(sys.exc_info()[0], sys.exc_info()[1], sys.exc_info()[2])
+        add_to_msg(pt2)
+        return("")
+
+    if output:
+        return(output)
+    else:
+        return("")
+
+
+def ck_cmd(cmd_str):
+    """
+    Check this command string for nasty stuff
+    if nasty stuff exists:
+        return True
+        else False
+    """
+    global nasty_list
+
+    for item in nasty_list:
+        if item in cmd_str:
+            key_word = item
+            debugg("comparing {} with {}".format(item, cmd_str))
+            if key_word == "drop" and "drop restore" in cmd_str:
+                return(False)
+            else:
+                return(True, key_word)
+
+    return(False, None)
+
+
+# ==============================================================================
+# =================================== MAIN =====================================
+# ==============================================================================
+
+def main ():
+    """
+    Run a sql command string
+    """
+    global msg
+    global default_refname
+    global debugme
+    global db_home_name
+    global cru_domain
+    global dr_domain
+    global affirm
+    global cur
+    is_rac = None
+    ignore_err_flag = False
+    return_values = []
+
+    # Name to call facts dictionary being passed back to Ansible
```

Comment:
Is this a leftover comment?

Reply:
No, I had to move the code and forgot the comment. I've removed the comment.